examples/simplest.yaml

```diff
@@ -2,3 +2,8 @@ apiVersion: jaegertracing.io/v1
 kind: Jaeger
 metadata:
   name: simplest
+spec:
+  allInOne:
+    image: jaegertracing/all-in-one:latest
+  agent:
+    image: jaegertracing/jaeger-agent:latest
```

Comment:
@jpkrohling Is there any specific reason to include image tags on this template? 
Some tests are failing in IBM, since jaegertracing/all-in-one:latest do not have P and Z arch images.

Reply:
What should be used instead?

Reply:
@jpkrohling we can leave it blank. Jaeger operator will update (based on the operator version)
If it is ok, I can create PR to remove changes on this file.

Reply:
Those are example files, they can't be left blank. Can't they adapt this before running the test?